### ISSUE TITLE
Cannot access Watchdog on pico.

### ISSUE THREAD

**Issue by stuartdd** [arduino/ArduinoCore-mbed#219]
Hi.

Been trying various ways to set up a watchdog. 

From my searches I have come to the following import:
`#include <Watchdog.h>`

This works but I cannot get access to the Watchdog as described in the code. The example given is as follows:

File:ArduinoCore-mbed-master/cores/arduino/mbed/drivers/include/drivers/Watchdog.h
Code example from the file:
```
* @code
 * Watchdog &watchdog = Watchdog::get_instance();
 * watchdog.start(500);
 *
 * while (true) {
      // kick watchdog regularly within provided timeout
      watchdog.kick();
      // Application code
 * }
 * @endcode
```
Error:'Watchdog' was not declared in this scope

I notice the the 'namespace' is 'mbed' so I have tried prefixing as follows 'mbed.Watchdog'. Is it that simple because this does not work either.

Is this the correct Watchdog setup procedure. I just need a Watchdog, any Watchdog will do!

Regards

Stuart

**Comment by stuartdd** [comment]
Found a solution but still have some questions.

I needed to `#include <mbed.h>`

Now I can do ` mbed::Watchdog::get_instance().start();`

and `mbed::Watchdog::get_instance().kick();`

This works but no matter what value I pass to start() it always resets in about 8 seconds!

Also `mbed::Watchdog::watchdog.get_timeout();` always returns 0;

 `mbed::Watchdog::watchdog.get_max_timeout();` returns 8388607.

I can also do:
`mbed::Watchdog &watchdog = mbed::Watchdog::get_instance();` 
in the declaration section of my code (outside of setup())

Then:`watchdog.start(30000);`
and `watchdog.kick();`

But it does not change the time out from approx 8 seconds.

Regards
